.safety-policy.yml

```
@@ -0,0 +1,18 @@
+  cvss_severity:
+    - critical
+    - high
+    
+# Optional: Ignore specific vulnerabilities by ID
+# ignore_vulnerabilities:
+#   - vulnerability_id: "51457"
+#     reason: "False positive - not applicable to our use case"
+
+# Report settings
+report:
+  json_output: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)